docs/pages/application-access/okta/reference.mdx

```diff
@@ -0,0 +1,144 @@
+---
+title: Okta Service Reference Documentation
+description: Configuration and CLI reference documentation for Teleport Okta service.
+---
+
+## Configuration
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+The following snippet shows the full YAML configuration of an Okta Service
+appearing in the `teleport.yaml` configuration file:
+
+```yaml
+(!docs/pages/includes/config-reference/okta-service.yaml!)
+```
+
+## Okta import rule resources
+
+Full YAML spec of Okta import rule resources managed by `tctl` resource commands:
+
+```yaml
+kind: okta_import_rule
+version: v1
+metadata:
+  name: test-rule
+  description: "Okta import rule for admins"
+spec:
+  # Mappings describes a list of match directives that describe what lavels to
+  # apply to specific Okta groups and applications that will be synchronized
+  # into Teleport.
+  mappings:
+  - match:
+    # Individual application IDs can be used to create matches. Imported Okta
+    # applications will be matched based on the app IDs listed here.
+    - app_ids: ["app1", "app2"]
+    # These labels will be added to matching applications.
+    add_labels:
+      app_label: app_label_value
+  - match:
+    # Similarly, group IDs can be used to create matches.
+    - group_ids: ["group1", "group2"]
+    # These labels will be added to the groups.
+    add_labels:
+      label1: value1
+```
+
+You can create a new `okta_import_rule` resource by running the following commands, which
+assume that you have created a YAML file called `okta-import-rule.yaml` with your configuration:
+
+<ScopedBlock scope={["enterprise"]}>
```

Comment:
I think we can remove the `ScopedBlock` tags here. Users tend not to realize that you can adjust scope, and since the default scope is `oss`, the code snippet won't appear for most users.